File: examples/SKYSAT_GEN-A_PUBLIC_ORTHO_MULTISPECTRAL.js
var dataset = ee.ImageCollection('SKYSAT/GEN-A/PUBLIC/ORTHO/MULTISPECTRAL');
var falseColor = dataset.select(['N', 'G', 'B']);
var falseColorVis = {
  min: 200.0,
  max: 6000.0,
};
Map.setCenter(-70.892, 41.6555, 15);
Map.addLayer(falseColor, falseColorVis, 'False Color');

File: examples/SKYSAT/SKYSAT_GEN-A_PUBLIC_ORTHO_MULTISPECTRAL.py
dataset = ee.ImageCollection('SKYSAT/GEN-A/PUBLIC/ORTHO/MULTISPECTRAL')

false_color = dataset.select(['N', 'G', 'B'])

false_color_vis = {'min': 200.0, 'max': 6000.0}

m = geemap.Map()
m.set_center(-70.892, 41.6555, 15)
m.add_layer(false_color, false_color_vis, 'False Color')
m
